ros_buildfarm/templates/release/release_create_trigger_task.Dockerfile.em

```diff
@@ -1,6 +1,7 @@
 # generated from @template_name
 
-FROM ubuntu:xenial
+@{os_code_name = 'focal'}@
+FROM ubuntu:focal
```

Comment:
Did you mean?
```suggestion
FROM ubuntu:@os_code_name
```

Reply:
Hah. Ayup I did. My first pass at these updates involved just changing the hard-coded value and it was only later I created the in-template variable to make future changes easier. I clearly missed the fact that I'd already "touched" this line as I was scanning to replace `xenial` with the variable.